screenpipe-audio/src/stt.rs

```diff
@@ -202,13 +201,22 @@ pub async fn stt(
     let mut total_frames = 0;
     let mut speech_frame_count = 0;
 
+    let mut  noise = 0.;
+
     for chunk in audio_data.chunks(frame_size) {
         total_frames += 1;
-        match vad_engine.is_voice_segment(chunk) {
-            Ok(is_voice) => {
-                if is_voice {
-                    speech_frames.extend_from_slice(chunk);
-                    speech_frame_count += 1;
+        match vad_engine.audio_type(chunk) {
```

Comment:
good!

my only concern is if it does not cause any issue with other OSes (windows mostly) although i dont see why it would be

Reply:
What concerns do you have?